workloads/files/workload-network-script-cm.yml

```diff
@@ -0,0 +1,961 @@
+      retries=0
+      while [ $retries -le 60 ] ; do
+        remaining_projects=`oc get projects | egrep "^network[0-9]+ " | wc -l`
+        echo "$(date -u) Deleting network0 project..."
+        if [ $remaining_projects -eq 0 ]; then
+          break
+        fi
+        sleep 1
+        retries=$[$retries + 1]
+      done
+      [ ${remaining_projects} -gt 0 ] && echo "$(date -u) Failed to clean up network0 project" && exit 1
+
+      pbench-copy-results --prefix ${NETWORK_TEST_PREFIX}
+      echo "$(date -u) Completed Network benchmarks with ${pair_count} pairs"
+    done
+    echo "$(date -u) Completed Network benchmarks"
+    # End of Test Code
+  p2p-network.yaml.template: |
+    provider: local
+    ClusterLoader:
+      cleanup: false
+      projects:
+      - num: 1
+        basename: network
+        ifexists: delete
+        templates:
+        - num: 1
+          file: /tmp/pbench-ssh.yaml
+        - num: ${pair_count}
+          file: /tmp/pod-rc-template.yaml
+          parameters:
+          - NS_VALUE: "server"
+        - num: ${pair_count}
+          file: /tmp/pod-rc-template.yaml
+          parameters:
+          - NS_VALUE: "client"
+  s2s-network.yaml.template: |
+    provider: local
+    ClusterLoader:
+      cleanup: false
+      projects:
+      - num: 1
+        basename: network
+        ifexists: delete
+        templates:
+        - num: 1
+          file: /tmp/pbench-ssh.yaml
+        - num: ${pair_count}
+          file: /tmp/svc-template.yaml
+          parameters:
+          - ROLE: "server"
+        - num: ${pair_count}
+          file: /tmp/pod-rc-template.yaml
+          parameters:
+          - NS_VALUE: "server"
+        - num: ${pair_count}
+          file: /tmp/svc-template.yaml
+          parameters:
+          - ROLE: "client"
+        - num: ${pair_count}
+          file: /tmp/pod-rc-template.yaml
+          parameters:
+          - NS_VALUE: "client"
+  network-scc.yaml.template: |
+    ---
+    kind: SecurityContextConstraints
+    apiVersion: v1
+    metadata:
+      name: uperf
+    priority:
+    allowPrivilegedContainer: ${NETWORK_TEST_HOSTNETWORK}
+    defaultAddCapabilities:
+    allowedCapabilities:
+    allowHostDirVolumePlugin: false
+    allowHostNetwork: ${NETWORK_TEST_HOSTNETWORK}
+    allowHostPorts: false
+    allowHostPID: false
+    allowHostIPC: false
+    seLinuxContext:
+      type: RunAsAny
+    runAsUser:
+      type: RunAsAny
+    supplementalGroups:
+      type: RunAsAny
+    fsGroup:
+      type: RunAsAny
+    groups:
+    - system:authenticated
+  pod-rc-template.yaml.template: |
+    ---
+    kind: Template
+    apiVersion: v1
+    metadata:
+      name: uperf
+      annotations:
+        description: A template for creating an uperf replicationcontroller
+        tags: uperf,perf
+    labels:
+      template: uperf
+    objects:
+    - apiVersion: v1
+      kind: ReplicationController
+      metadata:
+        labels:
+          name: "\${NS_VALUE}"
```

Comment:
I think we should avoid doing things like this.  When you start escaping `$`s in templates, something is wrong, probably the overall design.

Reply:
Good point, I will simplify this template such that it is only an OpenShift template rather than combined env var applied template and OpenShift template.